### PR TITLE
test: cypress - added tests for Document Viewer

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
added tests for document viewer

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts [hunk 1]
```diff
@@ -4,14 +4,27 @@ import {
   encodedXlsxDoc,
   encodedXlsDoc,
 } from "../../../../../fixtures/exampleEncodedDocs";
+const ppt = "https://ssz.sgp1.digitaloceanspaces.com/3ZEO2582C29EA0KKK2/ppt-on-population-pptxafa26c44-208f-46a3-89cc-8a5c020b6863.pptx"
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts:7]: can we fixtures to do this?

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts [hunk 2]
```diff
@@ -4,14 +4,27 @@ import {
...
 const ee = ObjectsRegistry.EntityExplorer,
   locator = ObjectsRegistry.CommonLocators,
   deployMode = ObjectsRegistry.DeployMode,
-  propPane = ObjectsRegistry.PropertyPane;
+  propPane = ObjectsRegistry.PropertyPane,
+  agHelper = ObjectsRegistry.AggregateHelper;
 
 describe("DocumentViewer Widget Functionality", () => {
   it("1. Add new DocumentViewer and verify in canvas", () => {
+    ee.DragDropWidgetNVerify("filepickerwidgetv2", 400, 900);
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts:20]: can we move test 1 to before and combine 2 and 3 as one case?

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts [hunk 3]
```diff
@@ -114,4 +143,77 @@ describe("DocumentViewer Widget Functionality", () => {
...
+
+    // Verify by entering link
+    propPane.UpdatePropertyFieldValue("Document link", ppt);
+    deployMode.DeployApp();
+    agHelper.AssertElementExist(locator._widgetInDeployed("documentviewerwidget"));
+    agHelper.AssertContains("Current file type is not supported", "not.exist");
+    deployMode.NavigateBacktoEditor();
+  })
+
+  it("9. Should not show an image file", () => {
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts:168]: can 9, 10 and 11 be combined? Any implications in doing so?

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts (hunk 2 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/DocumentViewer/DocumentViewer_spec.ts]: this test is meant to be tested with url, hence we are using this. We will repalce these urls with TED urls, facing some issues will udpate once resolved

**Review by sharanya-appsmith — APPROVED** [review]

**Pull request merged**